<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="GenButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SequenceEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PromptWords">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;请输入先序序列：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="unableThreaded">
       <property name="text">
        <string>关闭线索化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preThreaded">
       <property name="text">
        <string>先序线索化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inThreaded">
       <property name="text">
        <string>中序线索化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="postThreaded">
       <property name="text">
        <string>后序线索化</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="preTraverse">
       <property name="text">
        <string>先序遍历</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inTraverse">
       <property name="text">
        <string>中序遍历</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="postTraverse">
       <property name="text">
        <string>后序遍历</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ThrTraverse">
       <property name="text">
        <string>线索二叉树遍历</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="prompt_Input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>*输入限制：节点数据为单个字符，
空节点用‘#’表示。示例：
ABDH##I##EJ##K##CFL##M##GN##O
注：按住左键悬停在节点上方可查看节点信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="prompt_PreNPost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>*红色曲线指向前驱，
紫色曲线指向后继</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
